wao/scalping_strategy_tester.py

```diff
@@ -52,7 +52,8 @@ def get_year_range() -> str:
 def write_to_csv(list_of_row):
     print("write_to_csv:... ")
     column_title = ['coin', 'Brain', 'human_readable_time', 'timeframe', 'win_rate_percentage_per_day',
-                    'number_of_trades_per_day', 'average_percentage_per_trade', 'cumulative_percentage_per_day']
+                    'number_of_trades_per_day', 'average_percentage_per_trade', 'cumulative_percentage_per_day',
+                    'win_rate_percentage_per_year' if time_range == '1m' else 'win_rate_percentage_per_three_years']
```

Comment:
Wrong logic, there can be three years test with 1m 

Please use the hardcoded string suggested